.travis.yml

```diff
@@ -33,7 +33,10 @@ cache:
 before_script:
   - 'export DISPLAY=:99.0'
   - 'sh -e /etc/init.d/xvfb start'
-  - sleep 3
+  - 'sleep 3'
+
+script:
+  - 'npm run docs:generate'
```

Comment:
This should be moved to the `test` script in `package.json` so other test things (like linting) will run as expected